### PR TITLE
Rename EXT_feature_metadata → EXT_mesh_features

### PR DESCRIPTION
**Pull request by donmccurdy**
Following discussion in https://github.com/CesiumGS/glTF/pull/17. 

Will likely want some additional rephrasing, beyond the find/replace here, but will do that in a separate PR.

### CODE REVIEW

File: extensions/2.0/Vendor/EXT_mesh_features/schema/primitive.EXT_feature_metadata.schema.json [hunk 1]
```diff
@@ -0,0 +1,50 @@
+{
```
Comment (ptrgags) [inline extensions/2.0/Vendor/EXT_mesh_features/schema/primitive.EXT_feature_metadata.schema.json:1]: Make sure to update the filename of the JSON file, it still uses `EXT_feature_metadata`. There are a couple other files like this.

**Review by ptrgags — COMMENTED** [review]
@donmccurdy noticed a couple filenames that need to be updated.

Also as we talked about in Slack, you might need to rebase your branch to pick up recent changes, I notice the diff only does additions, not renames.

**Comment by donmccurdy** [comment]
Hm this was just a folder rename with a couple small changes before the `3d-tiles-next-rev` updates... I can't rebase it easily, so let me just refresh this as a new PR: https://github.com/CesiumGS/glTF/pull/20.

**Pull request closed without merging**